README.md

```diff
@@ -54,14 +54,16 @@ The next step for setup involves editing your `~/.bashrc` file and adding the
 following lines:
 
 ```
-export WORKON_HOME=~/.virtualenvs
-export PROJECT_HOME=~/projects
-export PIP_VIRTUALENV_BASE=~/.virtualenvs
-source /usr/local/bin/virtualenvwrapper.sh
+if [ -d "~/.virtualenvs" ]; then
```

Comment:
I understand why you want this here but I don't think it is needed. If you are choosing to work with virtualenv and using this approach then it would make sense that you have the directory already created. If you're worried about cross-environment syncing then you should be careful to bootstrap the environment so this doesn't error. Ultimately I want this repo to be about "how" to implement this and not about special cases.